polaris-react/src/components/Banner/Banner.tsx

```diff
@@ -124,24 +122,25 @@ export const Banner = forwardRef<BannerHandles, BannerProps>(function Banner(
 
   const actionMarkup =
     action || secondaryAction ? (
-      <div className={styles.Actions}>
+      <Box
+        paddingTop={withinContentContainer ? '3' : '4'}
+        paddingBottom={withinContentContainer ? '1' : undefined}
+      >
         <ButtonGroup>
           {primaryActionMarkup}
           {secondaryActionMarkup}
         </ButtonGroup>
-      </div>
+      </Box>
     ) : null;
 
   let contentMarkup: React.ReactNode = null;
-  let contentID: string | undefined;
 
   if (children || actionMarkup) {
-    contentID = `${id}Content`;
     contentMarkup = (
-      <div className={styles.Content} id={contentID}>
+      <Box paddingTop="05" paddingBottom="05">
```

Comment:
Wondering if we could do something like this until we can remove id:
```suggestion
      <Box paddingTop="05" paddingBottom="05">
        <div id={contentID} />
```

We'd have to ping the person who contributed the id prop to the component to see if this would be sufficient

Reply:
Might be some context here https://github.com/Shopify/polaris/pull/3199